docs/ReleaseNotes.md

```diff
@@ -23,6 +23,7 @@ The included licenses apply to the following files:
 - Fixed regression: [#7508](https://github.com/microsoft/DirectXShaderCompiler/issues/7508) crash when calling `Load` with `status`.
 - Header file `dxcpix.h` was added to the release package.
 - Moved Linear Algebra (Cooperative Vector) DXIL Opcodes to experimental Shader Model 6.10
+- Added `-fhlsl-unused-resource-bindings=<value>` an option to allow deciding on how to treat unused resource bindings in DXIL; `strip` (default), `keep` or `reserve`. `strip` will strip unused resources before generating bindings for resources without a `: register`, `keep` keeps them around while marking them with `D3D_SIF_UNUSED` in reflection and `reserve` will keep them reserved for generated bindings while stripping them afterwards. See [explanation](https://github.com/microsoft/DirectXShaderCompiler/pull/7643#issuecomment-3496917202) for more details.
```

Comment:
```suggestion
- Added `-fhlsl-unused-resource-bindings=<value>` an option to allow deciding on how to treat unused
  resource bindings in DXIL; `strip` (default), `keep-all` or `reserve-all`. `strip` will strip all unused resources,
  `keep-all` will keep them around while marking them with `D3D_SIF_UNUSED` in reflection, and `reserve` will keep
  their binding slots reserved when implicit bindings are assigned but not include them in the shader binding table
  or reflection. See [explanation] https://github.com/microsoft/DirectXShaderCompiler/pull/7643#issuecomment-3496917202)
  for more details.```

Reply:
Actually, just add the text for `reserve-all` for now. The `keep-all` should be added in the follow-up change.